.idea/modules.xml

```diff
@@ -4,6 +4,7 @@
     <modules>
       <module fileurl="file://$PROJECT_DIR$/app/app.iml" filepath="$PROJECT_DIR$/app/app.iml" />
       <module fileurl="file://$PROJECT_DIR$/measurement-kit-android-sample.iml" filepath="$PROJECT_DIR$/measurement-kit-android-sample.iml" />
+      <module fileurl="file://$PROJECT_DIR$/measurement_kit/measurement_kit.iml" filepath="$PROJECT_DIR$/measurement_kit/measurement_kit.iml" />
```

Comment:
I searched in all the settings in .idea folder and removed all the references to that folder but it keeps recreating it. I think that with another project we should not have this problem.